app/controllers/questions/notification_preference_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Questions
+  class NotificationPreferenceController < QuestionsController
+    private
```

Comment:
these methods aren't private on our other questions controllers - let's try to be consistent by either making them all private or not

Reply:
Ah yeah, consistency is good. I think it's a general Rails idiom to make non-action methods in controllers `private` (e.g. anything except new/index/edit/etc.), so I'd advocate for the `private` version in general. But since that's a refactor/computer gardening, it seems good to just make this non-private.

Reply:
That's a good point Tom, I'll put a chore in the backlog for converting all non-action methods on controllers to private methods.

Reply:
✅